decidim-elections/app/views/decidim/elections/admin/census/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= append_javascript_pack_tag "decidim_elections_admin" %>
+
+<%= render "decidim/elections/admin/elections/tabs_menu" do %>
+  <%= button_tag t("save", scope: "decidim.elections.admin.elections.form"), class: "main-tabs-menu__cta-button #{"hide" unless election.census}", form: "census-election-form" %>
+<% end %>
+
+<div class="item_show__header form-defaults">
+  <h1 class="item_show__header-title">
+    <%= t(".title") %>
+  </h1>
+  <%= select_tag "census_manifest",
+                 options_for_select(census_manifests.to_h { |manifest| [manifest.label, manifest.name] }, selected: election.census&.name),
+                 include_blank: t(".choose_census"),
+                 id: "census-manifest-selector" %>
+
+</div>
+  <% if election.census_ready? %>
+   <% announcement = t("decidim.elections.censuses.token_csv.uploaded_data", election_title: translated_attribute(election.title)) %>
+   <% announcement += "<br>" + t(".voters_count", count: "<strong>#{census_count}</strong>") %>
```

Comment:
I have an issue with this implementation. Maybe for Europe works "Uploded data Blah, blah<br> voters count 20", but maybe there are other languages that we do not know where this doesn't work. I would to the following: 

- rename the key to : "uploaded_data_html", and bind to it the following things: count + election title

```suggestion
   <% announcement = t("decidim.elections.censuses.token_csv.uploaded_data_html", election_title: translated_attribute(election.title), count: "<strong>#{census_count}</strong>") %>
```
Also, i think that we should use `sanitize_translated(election.title)` instead of `translated_attribute`